script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -280,24 +280,24 @@ The woman walked away as if it was a matter of course.
 <Page77>
 "...See you tomorrow."
 Tomorrow we can talk again, just like today.
-I'm glad. For the first time since I woke up from the accident, I felt human again.
+I was glad. For the first time since I woke up from the accident, I felt human again.
 <Page78>
 It became my routine to go to the field in the afternoon.
 The woman would get upset if I called her Aoko. She said she didn't like her name.
 After much pondering, I decided to call her "Sensei" because she seemed to be an important person.
 <Page79>
-Sensei listens to me seriously no matter what I say and solves my problems with just a few words.
+Sensei listened to me seriously no matter what I said and solved my problems with just a few words.
 <Page80>
 After the accident, I had become uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
 <Page81>
-Only when I was in this field did my anxiety disappear because I knew that I can be my normal self here.
-Even though I don't know who she is, she might really be a teacher in some school.
+Only when I was in this field did my anxiety disappear because I knew that I could be my normal self here.
+Even though I didn't know who she was, she might really be a teacher in some school.
 <Page82>
-But that's just a "what if" that has nothing to do with me.
-I enjoy being with Sensei.
-It's that simple, that's all that matters.
+But that was just a "what if" that had nothing to do with me.
+I enjoyed being with Sensei.
+It was that simple, that was all that mattered.
 <Page83>
-"Hey, Sensei. Look what I can do."
+"Hey, Sensei. Look I can do this."
```

Comment:
Hmm, prefer original here. Has more of a "kid showing off" vibe.

Reply:
I think both work fine, 2nd sentence also has that kid vibe in the form where kids go like "LOOK LOOK I CAN DO THIS!!"